spec/index.bs

```diff
@@ -6176,6 +6347,20 @@ dictionary GPURenderPassDescriptor : GPUObjectDescriptorBase {
     Issue(gpuweb/gpuweb#503): support for no attachments
 </div>
 
+<div algorithm="GPURenderPassDescriptor accessors" dfn-for=RenderPassDescriptor>
+    For a given {{GPURenderPassDescriptor}} value |descriptor|, the syntax:
+
+      - |descriptor|.`renderExtent` refers to
```

Comment:
I think we should refer to this with internal-slot syntax and possibly give it a dfn. E.g.

```
      - |descriptor|.<dfn dfn for=GPURenderPassDescriptor>\[[renderExtent]]</dfn> refers to
```

Reply:
This is what I had, but it complains about an unused dfn. So there is an Issue right below about this.